tests/DevApps/AzureFunctions/Function1.cs

```diff
@@ -0,0 +1,24 @@
+using Microsoft.AspNetCore.Http;
```

Comment:
add the license info to all the classes
// Copyright (c) Microsoft Corporation. All rights reserved.
// Licensed under the MIT License.